FlashDevelop/Settings/Accessors.cs

```diff
@@ -608,6 +608,16 @@ public Int32 MaxRecentFiles
             set { this.maxRecentFiles = value; }
         }
 
+        [DefaultValue(1000)]
+        [DisplayName("Max Trace Lines")]
+        [LocalizedCategory("FlashDevelop.Category.Features")]
+        [LocalizedDescription("FlashDevelop.Description.MaxTraceLines")]
```

Comment:
You should add this key into localization files.

Reply:
ok, updated.